### DISCUSSION TITLE
keeping npm package versions in sync across multiple repositories

### DISCUSSION THREAD

**Discussion by marco4320** [renovatebot/renovate#25161] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

36.109.4

### If you're self-hosting Renovate, select which platform you are using.

Bitbucket Server

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Hi, 
we have a TypeScript application that is split across multiple git repositories (Bitbucket). For example, common functionality is in one repo, and the app code in another. Renovate has been enabled on both repos and provides suggestions on version updates in package.json (hence, working as expected). 

I am new to renovate. Now, what I would like to achieve is to ensure that the npm package versions are kept in sync across both repositories. Let's say, if I am using "@aws-sdk/client-ec2": "3.427.0" in the repo with the common functionality (our "source of truth"), then and I want to make sure that the same version is used in the app repo as well. However, this may not always be the latest version available. Essentially, one "master" repo provides versions for certain packages, and the other ones are to follow.

Is something like this actually supported by renovate? Any hint would be appreciated.

Many thanks, 
Marco

### What you tried so far.

Running renovate in both repositories. Works as expected. However, the package versions may not be in sync across multiple repositories. Manually fixing version numbers using package rules I do not consider as an option.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
I'm not aware of any solution for this. You would need to have a custom registry or datasource to enforce a particular version like that
